src/trebble.js

```diff
@@ -249,31 +260,49 @@ const fieldsToMask = [
   "creditScore",
 ];
 
+/**
+ * Generates an object of fields to mask.
+ *
+ * We'll use an object because it's faster to check if a key exists in an object,
+ * then it is to check if the key exists in an array.
+ *
+ * @param {string[]?} additionalFieldsToMask
+ * @returns {object}
+ */
+function generateFieldsToMask(additionalFieldsToMask = []) {
+  const fields = [...fieldsToMask, ...additionalFieldsToMask];
+  const fieldsMap = fields.reduce((acc, field) => {
+    acc[field] = true
+    return acc
+  }, {});
+  return fieldsMap;
+}
```

Comment:
Converted everything to an object for faster lookup - this should reduce overhead per request, if users have large payloads and/or a lot of fields they want to mask